<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SfMainWindow</class>
 <widget class="QMainWindow" name="SfMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="TF" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>200</width>
      <height>755</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="drawBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>绘制</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
     </property>
     <property name="text">
      <string>曲面绘制模块</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>u向B样条类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>准均匀B样条</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>均匀B样条</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>u向B样条类型</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>u向曲线次数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le2"/>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="cb2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>准均匀B样条</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>均匀B样条</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="clcBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清屏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>711</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>711</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>读取</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>u向曲线次数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le1"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="picBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>截屏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MF" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>571</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>曲面绘制</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
